app/src/main/java/org/mifos/mobile/ui/fragments/base/BaseFragment.java

```diff
@@ -73,6 +74,15 @@ public void onAttach(Context context) {
         }
     }
 
+    public void replaceFragment(Fragment fragment) {
```

Comment:
You can the replaceFragment function of BaseActivity. This separate function is not needed.